init-container/CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Running the init container locally
+
+1. Make sure you have a working Kubernetes cluster (run `kubectl get pods` to check).
+2. Start by running the decryptor API - run the following in the decryptor folder (`src/decrypt-api`):
+```
+dotnet run
+```
+3. Run the following command to get a valid service account token:
+```
+kubectl get secret $(kubectl get sa default -o jsonpath='{.secrets[0].name}') -o jsonpath='{.data.token}' | base64 -D > token.txt
+```
+4. Set the following env var for the init container to work correctly:
+```
+export set TOKEN_FILE_PATH=$(pwd)/token.txt
+export set KAMUS_URL=http://localhost:5000
+```
+5. Now you can run the init container:
+```
+node index.js -e encrypted -d decrypted -n output.json 
```

Comment:
Explain what is `encrypted` ? is a folder? is it a file?

Reply:
it's a folder - I put it there with some ready-made encrypted content that is ready for debugging

Reply:
I mean, it needs to be more clear in the MD.